src/db_process.c

```diff
@@ -312,6 +350,11 @@ static int sym_info_count_query_callback(void *count, int argc, char **argv, cha
 
 int vm_info_count(sqlite3 *db) 
 {
+	// First check if the vm table exists
+	if (0 == db_table_exists(db, "vm")) {
+		errx(1, "vm table does not exist on db %s", dbname);
+	}
```

Comment:
I'd be tempted to create a wrapper function or macro long the lines of db_assert_table_exists() that does what this block does.

Reply:
Thank you, added! Please review :-)